pkg/miscellaneous/miscellaneous.go

```diff
@@ -28,16 +30,25 @@ func GetLatestFullSnapshotAndDeltaSnapList(store snapstore.SnapStore) (*snapstor
 		return nil, nil, err
 	}
 
+	metrics.SnapstoreLatestDeltasTotal.With(prometheus.Labels{}).Set(0)
```

Comment:
Why are you resetting it to zero. This add additional value point to time series. Not expected.